### PR TITLE
Functions generators

### PR DESCRIPTION
**Pull request by zouxuoz**

### CODE REVIEW

File: templates/functions/webhook/handler.ts.ejs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+type WebhookResult = {
+  data: {
+    result: string,
+  },
+};
+
+export default async (event: any, ctx: any) : Promise<WebhookResult> => {
+  return {
+    data: {
```
Comment (vorobeez) [inline templates/functions/webhook/handler.ts.ejs]: @zouxuoz Are you sure about return value format? I believe it should be like `{ statusCode: 200, body: JSON.stringify({}) }`.

File: src/engine/commands/generate/commands/trigger.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    .option("type", {
+      alias: "t",
+      describe: translations.i18n.t("generate_trigger_type_describe"),
+      type: "string",
+      choices: Object.values(TriggerType),
+    })
+    .option("operation", {
+      alias: "o",
+      describe: translations.i18n.t("generate_trigger_operation_describe"),
+      type: "string",
```
Comment (vorobeez) [inline src/engine/commands/generate/commands/trigger.ts]: @zouxuoz Why is there no choices for operation like for type?

Comment (zouxuoz) [reply src/engine/commands/generate/commands/trigger.ts]: @vorobeez because it could be `AnyTableName.(create|update|delete)`

Comment (vorobeez) [reply src/engine/commands/generate/commands/trigger.ts]: I see, ok

File: templates/functions/webhook/handler.ts.ejs (hunk 1 above)
Comment (zouxuoz) [reply templates/functions/webhook/handler.ts.ejs]: @vorobeez fixed

**Pull request merged**